modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -124,12 +128,19 @@ def format_representative(representative)
           }
         end
 
-        def validate_individual_poa_code!(poa_code)
-          return if ::Veteran::Service::Representative.where('? = ANY(poa_codes)', poa_code).any?
+        def requested_representative(poa_code, rep_id)
```

Comment:
I combined the poa code validation and registration number validation into one lookup. If we can't find a rep with the given poa code and registration number, then throw an error.

This returns the first result sorted by `created_at desc` which we'll need later on.